src/app/problems/json-editor/json-editor.component.ts

```diff
@@ -21,7 +21,8 @@ export class JsonEditorComponent implements OnInit {
 
     ngOnInit(): void {
         const element = document.getElementById('editor');
-
+        //change ID to make sure each editor is separated
+        element.id = 'editor-' + Math.floor(Math.random() * 1000);
```

Comment:
Why do we need this?
Also having random is not a good idea. It's better to have a deterministic solution.